src/features/nft/nftModals/likeListData.ts

```diff
@@ -1,24 +1,24 @@
 const likes = [
-  { id: 0, date: '20/11/21', userName: 'L TestName', sold: 1, followers: 4 },
-  { id: 1, date: '19/11/21', userName: 'C TestName', sold: 5, followers: 7 },
-  { id: 2, date: '18/11/21', userName: 'D TestName', sold: 7, followers: 2 },
-  { id: 3, date: '17/11/21', userName: 'E TestName', sold: 10, followers: 5 },
-  { id: 4, date: '16/11/21', userName: 'G TestName', sold: 9, followers: 9 },
-  { id: 5, date: '15/11/21', userName: 'F TestName', sold: 6, followers: 5 },
-  { id: 6, date: '14/11/21', userName: 'X TestName', sold: 1, followers: 3 },
-  { id: 7, date: '13/11/21', userName: 'Y TestName', sold: 3, followers: 2 },
-  { id: 8, date: '12/11/21', userName: 'I TestName', sold: 2, followers: 6 },
-  { id: 9, date: '11/11/21', userName: 'M TestName', sold: 5, followers: 6 },
-  { id: 10, date: '10/11/21', userName: 'K TestName', sold: 4, followers: 3 },
-  { id: 11, date: '9/11/21', userName: 'Z TestName', sold: 9, followers: 9 },
-  { id: 12, date: '8/11/21', userName: 'O TestName', sold: 8, followers: 1 },
-  { id: 13, date: '7/11/21', userName: 'N TestName', sold: 7, followers: 4 },
-  { id: 14, date: '6/11/21', userName: 'R TestName', sold: 3, followers: 3 },
-  { id: 15, date: '5/11/21', userName: 'X TestName', sold: 11, followers: 5 },
-  { id: 16, date: '4/11/21', userName: 'B TestName', sold: 6, followers: 1 },
-  { id: 17, date: '3/11/21', userName: 'V TestName', sold: 8, followers: 2 },
-  { id: 18, date: '2/11/21', userName: 'I TestName', sold: 4, followers: 2 },
-  { id: 19, date: '1/11/21', userName: 'E TestName', sold: 7, followers: 3 },
+  { id: 0, date: '1 min ago', member: 'L TestName', sold: 1, followers: 4 },
```

Comment:
For `date` field, I think we can use `datetime` and use `dayjs ` to show. Refer https://day.js.org/docs/en/display/from-now